person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'unknown', parent_permision: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permision = parent_permision
+    @rentals = []
   end
```

Comment:
- [ ] I got the attached errors when I tried creating a student. It happens that there is a typographical error in your `Nameable`. Here you have it as `Nameable` while the class itself is `Namable`. Kindly fix this so the error by making use of the correct name of your class.

- [ ] I also got an error about the file name. You have `require './nameable'` but actually the name of your nameable file is `namable`. you should require the file with the correct file name `require './namable'` to fix the error.
<img width="398" alt="chrome_zOwO52dCVV" src="https://user-images.githubusercontent.com/81468675/175774738-51c04957-825b-4a10-b55a-3d4e85add089.png">

<img width="410" alt="slack_C7TqCeQGLv" src="https://user-images.githubusercontent.com/81468675/175774739-0cd30eb0-a9d2-4a86-b9bc-babc9428e2e7.png">